test/integration/patients/worklist/worklist.js

```diff
@@ -1273,8 +1265,8 @@ context('worklist page', function() {
 
     cy
       .get('.tooltip')
-      .should('contain', formatDate(dayjs(filterDate).startOf('week'), 'MM/DD/YYYY'))
-      .should('contain', formatDate(dayjs(filterDate).endOf('week'), 'MM/DD/YYYY'));
+      .should('contain', formatDate(dayjs(testDate()).startOf('week'), 'MM/DD/YYYY'))
```

Comment:
this was a big issue.  `filterDate` is not relative to the current time, it's the selected time.  But the display in the tooltip _is_ relative to the current time